hooks/pre/30-upgrade.rb

```diff
@@ -65,8 +65,8 @@ def fail_and_exit(message)
 
   if Kafo::Helpers.module_enabled?(@kafo, 'katello') || @kafo.param('capsule', 'pulp').value
     upgrade_step :start_mongo
-    upgrade_step :start_httpd
     upgrade_step :migrate_pulp
+    upgrade_step :start_httpd
```

Comment:
Which made me realize another problem, we can't exclude httpd from the service stop, we have to have everything but mongo stopped for the migration.  So, I updated that.